acceptance/managers/image_manager.go

```diff
@@ -0,0 +1,176 @@
+// +build acceptance
+
+package managers
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	dockertypes "github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+
+	h "github.com/buildpacks/pack/testhelpers"
+)
+
+var DefaultDuration = 10 * time.Second
+
+type ImageManager struct {
+	testObject *testing.T
+	assert     h.AssertionManager
+	dockerCli  client.CommonAPIClient
+}
+
+func NewImageManager(t *testing.T, dockerCli client.CommonAPIClient) ImageManager {
+	return ImageManager{
+		testObject: t,
+		assert:     h.NewAssertionManager(t),
+		dockerCli:  dockerCli,
+	}
+}
+
+func (im ImageManager) CleanupImages(imageNames ...string) {
+	im.testObject.Helper()
+	err := h.DockerRmi(im.dockerCli, imageNames...)
+	if err != nil {
+		im.testObject.Logf("%s: Failed to remove image from %s", err, imageNames)
+	}
+}
+
+func (im ImageManager) Inspect(image string) (dockertypes.ImageInspect, error) {
```

Comment:
nit: Could this be `InspectLocal` (or something similar) as this will only work for images on the docker daemon

Reply:
addressed in ff6aa1d356a5e5c1cb5c03b69d86741519eb4a95